cluster/raft_query_endpoints.go

```diff
@@ -287,16 +287,19 @@ func (s *Raft) Query(ctx context.Context, req *cmd.QueryRequest) (*cmd.QueryResp
 	var err error
 	resp := &cmd.QueryResponse{}
 	backoff.Retry(func() error {
+		// always reset err from previous iteration just in case
+		err = nil
```

Comment:
**suggestion (non-blocking):** we can move `var err ` here instead of sitting to `nil`